std/string.d

```diff
@@ -244,12 +250,32 @@ inout(char)[] fromStringz(inout(char)* cString) @nogc @system pure nothrow {
     first $(D '\0') that it sees as the end of the string. If $(D s.empty) is
     $(D true), then a string containing only $(D '\0') is returned.
 
-    $(RED Important Note:) When passing a $(D char*) to a C function, and the C
+    $(PITFALL When passing a $(D char*) to a C function, and the C
     function keeps it around for any reason, make sure that you keep a
     reference to it in your D code. Otherwise, it may become invalid during a
     garbage collection cycle and cause a nasty bug when the C code tries to use
-    it.
+    it.)
   +/
+immutable(char)* toStringz(in string s) @trusted pure nothrow
```

Comment:
I explained this in my PR description.

"I rearranged the  toStringz  overloads to put the one with the simpler signature first. Then a user browsing down will see that simple one as the entry point and the other one as an alternative. A small thing that has a bigger source diff than it really is, but makes things a bit more readable for the first-time browser."